tests/unit/tasks/util/postcss.ts

```diff
@@ -66,35 +81,50 @@ registerSuite('tasks/util/postcss', {
 				assert.isTrue(postCssModulesStub.calledBefore(cssNanoModuleStub));
 			},
 			'auto prefixer browsers'() {
-				postCssUtil.createProcessors('', '');
+				postCssUtil.createProcessors({
+					packageJson: packageJsonStub(),
+					dest: '',
+					cwd: ''
+				});
 				assert.isTrue(postCssNextStub.calledOnce);
-				assert.isTrue(postCssNextStub.calledWithMatch({
+				const [ autoPrefixrConfig ] = postCssNextStub.firstCall.args;
+
+				assert.deepEqual(autoPrefixrConfig, {
 					features: {
 						autoprefixer: {
 							browsers: [
-								'last 2 versions',
-								'ie >= 11'
+							'last 2 versions',
+							'ie >= 11'
 							]
 						}
 					}
-				}));
+				});
+
 				assert.isTrue(cssNanoModuleStub.calledOnce);
-				assert.isTrue(cssNanoModuleStub.calledWithMatch({
+				assert.deepEqual(cssNanoModuleStub.firstCall.args[0], {
 					autoprefixer: false
-				}));
+				});
 			},
 			'generate scoped name': {
 				'generate localised scoped name for dev'() {
-					postCssUtil.createProcessors('', '', false);
-					assert.isTrue(postCssModulesStub.calledWithMatch({
-						generateScopedName: '[name]__[local]__[hash:base64:5]'
-					}));
+					postCssUtil.createProcessors({
+						packageJson: packageJsonStub(),
+						dest: '',
+						cwd: '',
+						dist: false
+					});
+					const [{generateScopedName}] = postCssModulesStub.firstCall.args;
+					assert.equal(generateScopedName, '[name]__[local]__[hash:base64:5]');
```

Comment:
I think this would read better throughout these changes as

```ts
assert.equal(postCssModulesStub.firstCall.calledWithMatch({
   generatedScopedName: '[name]__[local]__[hash:base64:5]' 
}));
```

The double destructuring can be confusing

Reply:
I think the only issue I had is it's not clear what `calledWithMatch` does (even when searching for docs) whereas I feel personally that the destructuring is quite intuitive and hopefully readable, as is the API to `assert.equal`? Also that I find the error logging with `assert.ok` to be not great because it just says stuff like expected false to be true or something, but I really like the errors you get with `assert.equals`

What do you think if we try it this way? If not I can change it